neo3/wallet/account.py

```diff
@@ -247,6 +264,34 @@ def private_key_to_nep2(private_key: bytes, passphrase: str) -> bytes:
 
         return encoded_nep2
 
+    @staticmethod
+    def private_key_from_wif(wif: str) -> bytes:
+        """
+        Decrypt a private key from a wif.
+
+        Args:
+            wif: the wif that will be decrypted.
+
+        Raises:
+            ValueError: if the wif is not valid.
+        """
+        try:
+            decoded_key: bytes = base58.b58decode_check(wif)
+        except Exception:
+            raise ValueError("Base58decode failure of wif")
+
+        if len(decoded_key) != 34:
+            raise ValueError(f"The decoded wif length should be {len(types.UInt160.zero()) + 1}, while the given wif "
```

Comment:
```suggestion
            raise ValueError(f"The decoded wif length should be 34, while the given wif "
```

Reply:
I'll change it to `{len(WIF_PREFIX) + len(types.UInt256.zero()) + len(WIF_SUFFIX)}` instead of `34` to be more self-explanatory.

Reply:
I like the idea, except that `len(types.UInt256.zero())` is not the right type. It does happen to have 32 bytes as length, but other than that the type has no relation with a private key. They're actually specifically used in mamba as follows
* UInt160 = to uniquely identify smart contracts and signing authorities (i.e. Consensus nodes and Transaction owners)
* UInt256 = for storing hashes that can uniquely identify Blocks and Transactions among other objects